files/bootstrap.sh

```diff
@@ -461,6 +468,9 @@ if [[ "$CONTAINER_RUNTIME" = "containerd" ]]; then
     sudo mkdir -p /etc/containerd
     sudo mkdir -p /etc/cni/net.d
     sudo sed -i s,SANDBOX_IMAGE,$PAUSE_CONTAINER,g /etc/eks/containerd/containerd-config.toml  
+    if [[ -n "$CONTAINERD_EXTRA_CONFIG_TOML" ]]; then
+        echo "$CONTAINERD_EXTRA_CONFIG_TOML" >> /etc/eks/containerd/containerd-config.toml
+    fi
```

Comment:
This block should be moved above L470, so that we have the ability to specify SANDBOX image in custom containerd config and get is replaced by the right pause container in the bootstrap.sh script.

Reply:
Makes sense, I'm going to change this and force-push.